### ISSUE TITLE
OSM menu

### ISSUE THREAD

**Issue by Hades1503** [osmandapp/OsmAnd-iOS#1622]
Moving the OSM menu to the settings list would be easier and more accessible
![image](https://user-images.githubusercontent.com/79785663/142609177-05e3f4f1-0607-44ae-9312-4e8af104fe73.jpeg)

**Comment by vshcherb** [comment]
Yes we need to have something similar to Android - https://docs.osmand.net/en/main@latest/osmand/start-with/main-menu#customize-android-advanced

**Comment by abdullahO2** [comment]
> Yes we need to have something similar to Android - https://docs.osmand.net/en/main@latest/osmand/start-with/main-menu#customize-android-advanced
update:
https://docs.osmand.net/docs/user/map/map-context-menu#customize
